core/src/autogluon/core/searcher/bayesopt/autogluon/searcher_factory.py

```diff
@@ -169,7 +169,7 @@ def gp_multifidelity_searcher_factory(**kwargs) -> GPMultiFidelitySearcher:
     :return: GPMultiFidelitySearcher object
 
     """
-    supp_schedulers = {'hyperband_stopping', 'hyperband_promotion'}
+    supp_schedulers = {'hyperband_stopping', 'hyperband_promotion', 'local'}
```

Comment:
Are you sure about this? Beware that searchers for multi-fidelity schedulers have a different behavior than searchers for fifo scheduler.

Reply:
Thanks for catching - removed.